### DISCUSSION TITLE
How to delete app icon on push message?

### DISCUSSION THREAD

**Discussion by whcho888** [invertase/react-native-firebase#4526] [Q&A]
How to delete app icon on push message?

**Comment by mikehardy** [comment]
Hi there! With apologies, I'm not sure what this even means. You'll need to be a lot more specific and include information about platform etc. Also note that we don't do much with notifications in this package, so if the upstream SDK (firebase-android-sdk / firebase-ios-sdk) does not let you do it in the notification payload, we likely won't have any support existing or planned here and you'll want a full-featured 3rd party device-local notification package
